### PR TITLE
LPD-23118 - Prevent user from submit the form if attribute name "src"

### PR DESCRIPTION
**Pull request by ilzamcmed**
Jira issue: https://liferay.atlassian.net/browse/LPD-23118

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
Investigating playwright errors.

File: modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/global-js/AttributeFields.tsx [hunk 1]
```diff
@@ -82,11 +82,11 @@ export default function AttributeFields({
 							}
 							else {
 								setErrorMessage(null);
-
-								onAttributeChange(index, {name: target.value});
 							}
+							onAttributeChange(index, {name: target.value});
```
Comment (ethib137) [inline modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/global-js/AttributeFields.tsx:86]: @ilzamcmed please add a line break before `onAttributeChange(index, {name: target.value});`

**Review by ethib137 — CHANGES_REQUESTED** [review]

File: modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/global-js/AttributeFields.tsx (hunk 1 above)
Comment (ilzamcmed) [reply modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/global-js/AttributeFields.tsx:87]: Done!

**Comment by ilzamcmed** [comment]
@ethib137, the playwright's failures are not related to the changes.

**Review by ethib137 — APPROVED** [review]
Thank you @ilzamcmed ! LGTM

**Comment by ethib137** [comment]
ci:forward:force

**Comment by ilzamcmed** [comment]
ci:reevaluate:1367401_5753

**Comment by ethib137** [comment]
ci:test:stable

**Comment by ilzamcmed** [comment]
I'm thinking this is something related to Analytics Cloud because of this path here:  `':dxp:apps:osb:osb-faro:osb-faro-mock-engine-client:compileJava'.`
I've sent a message to Marcos to see if he is aware of it.

**Comment by ethib137** [comment]
ci:test:stable

**Comment by ethib137** [comment]
ci:test:stable

**Comment by ethib137** [comment]
@ilzamcmed these failures are all unrelated. Can you manually forward this, please?

**Comment by ilzamcmed** [comment]
Sure @ethib137
Thanks!

**Comment by ilzamcmed** [comment]
Resent to https://github.com/brianchandotcom/liferay-portal/pull/149297

**Pull request closed without merging**